webapp/src/ts/effects/reports.effects.ts

```diff
@@ -224,4 +239,114 @@ export class ReportsEffects {
       }),
     );
   }, { dispatch: false });
+
+  launchEditFacilityDialog = createEffect(() => {
+    return this.actions$.pipe(
+      ofType(ReportActionList.launchEditFacilityDialog),
+      tap(() => {
+        const firstSelectedReportDoc = this.selectedReports && this.selectedReports[0]?.doc;
+        this.modalService
+          .show(EditReportComponent, { initialState: { model: { report: firstSelectedReportDoc } } })
+          .catch(() => {});
+      }),
+    );
+  }, { dispatch: false });
+
+  verifyReport = createEffect(() => {
+    return this.actions$.pipe(
+      ofType(ReportActionList.verifyReport),
+      tap(({ payload: { verified } }) => {
```

Comment:
I'm wondering if it's better to extract all these function definitions and checking on a service/utils and keep this effect straight forward. What do you think?

Reply:
I think this can totally be made better (hence the comment), but I wanted to keep it as close to the original code as possible because I want to minimize potential bugs. 
Would you mind if we leave it like this for now and refactor it completely in a later iteration?

Reply:
Okay! Let's add a ticket to pick up on later iteration :)

Reply:
Created: https://github.com/medic/angular10-migration/issues/42

I'll update this issue with links to the branch (once I merge this PR). 

I didn't create the issue in the `cht-core` repo because it felt weird to have an issue about some code that sits in a random branch. 
I think, at the end of the migration, and after we merge to `master`, we should evaluate every issue left in the migration repo and re-create them in the `cht-core` repo. Does that sound good?